src/serializer/serializer.js

```diff
@@ -257,7 +257,14 @@ export class Serializer {
       // the signature for generate is not complete, hence the any
       let generated = statistics.babelGenerate.measure(() => generate(ast, { sourceMaps: sourceMaps }, (code: any)));
 
-      invariant(!this.logger.hasErrors());
+      if (this.logger.hasErrors()) {
```

Comment:
With your other change to have format default to the empty string, you don't have to do this manual inlining here. Please remove the changes to the serializer.

Reply:
If we use the invariant method, `https://github.com/VictorHom/prepack/blob/ad9b7d11d08f151a71c958f5eaf4b21088698b12/src/invariant.js#L17`

will set the name of the error to be "Invariant Violation", so that will still be there. Is that alright?

Reply:
Not sure what you mean. This error indicates an internal Prepack error, which is an Invariant Violation.

Reply:
That makes sense. A misunderstanding from me on what invariant meant. I'll fix

Reply:
That makes sense. A misunderstanding from me on what invariant meant. I'll fix